app/v1/controllers/AmendPensionChargesController.scala

```diff
@@ -74,12 +81,14 @@ class AmendPensionChargesController @Inject()(val authService: EnrolmentsAuthSer
             Some(request.body),
             Some(responseWrapperWithHateoas)
           ))
-
           Ok(responseWrapperWithHateoas).withApiHeaders(responseWrapper.correlationId).as(MimeTypes.JSON)
 
         case Left(errorWrapper) =>
-          val correlationId = getCorrelationId(errorWrapper)
-          val result = errorResult(errorWrapper).withApiHeaders(correlationId)
+          val resCorrelationId = errorWrapper.correlationId
+          val result = errorResult(errorWrapper).withApiHeaders(resCorrelationId)
+          logger.info(
+            s"[${endpointLogContext.controllerName}][${endpointLogContext.endpointName}] - " +
+              s"Error response received with CorrelationId: $resCorrelationId")
 
           auditSubmission(createAuditDetails(
```

Comment:
Should use `resCorrelationId` for audiSubmission, also for all the other Controllers too

auditSubmission(createAuditDetails(
            rawData,
            result.header.status,
            `resCorrelationId`,